src/pages/ContentScripts/features/developer-hovercard-info/view.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import getGithubTheme from '../../../../helpers/get-github-theme';
+import optionsStorage, { HypercrxOptions, defaults } from '../../../../options-storage';
+import { useTranslation } from 'react-i18next';
+import '../../../../helpers/i18n';
+import { rocketLight, rocketDark } from './base64';
+
+interface OpenRankProps {
+  developerName: string;
+  openrank: string;
+}
+
+const OpenRankView: React.FC<OpenRankProps> = ({ developerName, openrank }) => {
+  const [theme, setTheme] = useState<'light' | 'dark'>(getGithubTheme() as 'light' | 'dark');
+  const [options, setOptions] = useState<HypercrxOptions>(defaults);
+  const { t, i18n } = useTranslation();
+  useEffect(() => {
+    (async function () {
+      setOptions(await optionsStorage.getAll());
+      i18n.changeLanguage(options.locale);
+    })();
+  }, [options.locale]);
```

Comment:
Do we need `useTranslation()` here? Remove it if it is unused.

Reply:
Yes, I should remove this unused code. already edited